setup.py

```diff
@@ -90,6 +92,15 @@
         'console_scripts': [
             'invenio = invenio_app.cli:cli',
         ],
+        'invenio_config.module': [
```

Comment:
This shouldn't be added here - it's for the instances like CDS/Zenodo etc to add this entry point, because there should only be one for any given Python environment